src/pages/viewProfile/index.js

```diff
@@ -0,0 +1,176 @@
+import { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import Button from "../../components/button";
+import "../../pages/viewProfile/viewProfile.css";
+import Card from "../../components/card";
+import ProfileCircle from "../../components/profileCircle";
+import { useParams } from "react-router-dom";
+import { get } from "../../service/apiClient";
+import jwt_decode from "jwt-decode";
+import ErrorMessage from "../../components/errorMessage";
+
+const initialState = {
+  id: "",
+  email: "",
+  role: "",
+  cohort_id: "",
+  firstName: "",
+  lastName: "",
+  biography: "",
+  githubUrl: "",
+  specialism: "",
+};
```

Comment:
Let's move this object to its own file to clean things up a bit. There's a `mockData.js` file in the `service` directory that could be used